docs/network/celo-sepolia/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Celo Sepolia Testnet
+description: Collection of resources to get started with Celo Sepolia Testnet (Celo's Developer Testnet).
+---
+
+# Celo Sepolia Testnet
+
+Get started with Celo Sepolia Testnet (Celo's Developer Testnet).
+
+---
+
+## What is the Celo Sepolia Testnet?
+
+The **Celo Sepolia Testnet** is Celo's new developer testnet built on Ethereum Sepolia, designed to replace Alfajores following the planned Holesky deprecation in September 2025. This testnet provides a stable environment for developers to build, test, and deploy decentralized applications before launching on Mainnet. Developers can access testnet funds via the faucet, interact with smart contracts, and experiment with wallets, dApps, and node operations without financial risk.
+
+**Key Details:**
+
+- **Chain ID**: 11142220
```

Comment:
this page seems very similar to [celo-sepolia-launch.md](https://github.com/celo-org/docs/pull/1994/files#diff-78dfd2df6cb416be5548861b64d00cbcdbfd09c00ebd58a79f5e7431b3f9679d)

Reply:
This one is intended to stay long term, whereas the notice page is intended to be a 1-pager we can easily share externally, with all information for the upcoming change in 1 place.